api_tests/test_endpoints.py

```diff
@@ -0,0 +1,37 @@
+import os
+import requests
+import json
+
+
+def run_this():
+    target_directory = os.path.join(os.getcwd(), "../json_files")
+    url = "http://0.0.0.0:5000/scores"
+
+    with open(
+        os.path.join(target_directory, "/sample_user_response.json")
```

Comment:
I'm pretty sure the "/" should be removed here so it looks like `os.path.join(target_directory, "sample_user_response.json")`